src/MUIDataTable.js

```diff
@@ -13,6 +13,8 @@ import classnames from 'classnames';
 import cloneDeep from 'lodash.clonedeep';
 import merge from 'lodash.merge';
 import isEqual from 'lodash.isequal';
+import find from 'lodash.find';
+import isUndefined from 'lodash.isUndefined';
```

Comment:
try changing this to:

import isUndefined from 'lodash.isundefined';

Reply:
Haha, that's crazy, no errors for me when running this locally, but nice catch, will fix.